### ISSUE TITLE
Status is ignored in CfnResponseSender

### ISSUE THREAD

**Issue by anastasiosl** [SunRun/cfn-response-java#1]
Hello,

I noticed that the passed in value for `Status` is ignored in `CfnResponseSender#sender` and it is always set to `SUCCESS`. Any particular reason why this is happening?

```
        // Compose response
        final CfnResponse<T> response = new CfnResponse<>();
        response.setData(data);
        response.setStatus(Status.SUCCESS);
        response.setLogicalResourceId(event.getLogicalResourceId());
        response.setPhysicalResourceId(physicalResourceId == null ? context.getLogStreamName() : physicalResourceId);
        response.setReason(reason == null ? "See the details in CloudWatch Log Stream: " + context.getLogStreamName() : reason);
        response.setRequestId(event.getRequestId());
        response.setStackId(event.getStackId());
```

I have the change and I can make a PR for this, however, I would like to know what's the reason for always responding with success to CloudFormation. I have just started using custom CFN resources with Lambda, so were there any issues when replying with failure? (I know the default timeout is ridiculous, like 90 minutes, but one can use a wait condition timeout to remedy that issue)

Thanks

**Comment by jhovell** [comment]
@anastasiosl nope, that's definitely a bug! A PR would be great... I'll release it right away. Thanks for spotting it  👀

**Comment by jhovell** [comment]
Replying with FAILURE is definitely a possible outcome. That tells Cfn to initiate a rollback

**Comment by jhovell** [comment]
@anastasiosl just released your fix to maven central as 1.0.1. It usually takes a few hours to show up in the repo, I'll close this issue once I see it is there. Thanks again!

**Comment by anastasiosl** [comment]
@jhovell thanks! By the way, it's there. I am using version 1.0.1 right now in my project. Closing this issue.
